pytest.ini

```diff
@@ -0,0 +1,7 @@
+[pytest]
+python_classes = *Test Test*
+python_files = test_* *_test
+python_functions = test_* *_test
+testpaths = tests
+addopts = -rsxX -l --tb=short
```

Comment:
Why using all those defaults ? half of them I don't remember exactly.

But I'm more sure we want by default to stop of first failure